cmd/notary-server/config.go

```diff
@@ -89,7 +89,12 @@ func getStore(configuration *viper.Viper, hRegister healthRegister) (
 		if err != nil {
 			return nil, err
 		}
-		sess, err = rethinkdb.Connection(storeConfig.CA, storeConfig.Source)
+		tlsOpts := tlsconfig.Options{
```

Comment:
Non-blocking:  I think if we add the cert and key to `TestGetStoreRethinkDBStoreConnectionFails` in `cmd/notary/main_test.go`, that might cover some of these uncovered lines, since the test is failing to parse due to not having a cert and key configured here.